src/lib/comms/sol-socket-impl-riot.c

```diff
@@ -68,14 +68,15 @@ ipv6_udp_recvmsg(struct sol_socket_riot *s, void *buf, size_t len, struct sol_ne
     memcpy(cliaddr->addr.in6, iphdr->src.u8, sizeof(iphdr->src.u8));
     cliaddr->port = byteorder_ntohs(udphdr->src_port);
 
-    copysize = pkt->size < len ? pkt->size : len;
-    memcpy(buf, pkt->data, copysize);
+    copysize = pkt->size < buf->capacity ? pkt->size : buf->capacity;
```

Comment:
exits only `sol_min()` in internal header,  i'm moving to sol-util.h as `sol_util_min()`